k8s.gcr.io/ensure-prod-gclb.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
Thanks - just made a similar change across the board in #233 - will extend that to use your `env` form